TSC/core-and-hosted-projects.md

```diff
@@ -0,0 +1,538 @@
+# Bytecode Alliance Core and Hosted Project Requirements
+
+This document enumerates the standards to which all Bytecode Alliance core and hosted projects are held. Additionally, it defines the process for applying for adoption as a hosted project or promotion from hosted project to core project.
+
+## What are Hosted and Core Projects?
+
+The Bytecode Alliance holds itself and its projects, whether hosted or core, to exceptional standards. This is part of our principle of [curation](https://github.com/bytecodealliance/governance/blob/main/operational-principles.md?plain=1#L17-L23) and we do not believe we can accomplish our [mission](https://github.com/bytecodealliance/governance/blob/main/mission.md) otherwise.
+
+Core projects are the Bytecode Alliance's flagships and are held to an even higher standard than hosted projects. In return, core projects are prominently advertised on the Bytecode Alliance webpage and can elect a delegate to the Bytecode Alliance's Technical Steering Committee, as per [the TSC charter](https://github.com/bytecodealliance/governance/blob/main/TSC/charter.md#composition).
+
+## Applying for Adoption as a Hosted Project
+
+If you would like to propose that the Bytecode Alliance adopt your project as a new hosted project, file an issue against [the Bytecode Alliance governance repository](https://github.com/bytecodealliance/governance). The issue should enumerate each item marked as "Hosted Projects: Required" with a checkbox. Provide supplemental information, examples, and evidence that your project meets the associated requirement as bullet points below this checkbox. Here is an example issue:
+
+```md
+Proposing the adoption of `my-example-project` as a Bytecode Alliance
+hosted project.
+
+My Example project is a cool project. Here is some high level information
+about what it does. There should be a link to the repo here.
+
+### Hosted Project Requirements
+
+* [ ] Alignment with the Bytecode Alliance Mission
+  * My Example Project has alignment with the Bytecode Alliance mission
+    because it is all about security, efficiency, and modularity blah
+    blah blah
+* [ ] Code Review
+  * We follow the code review requirements, for example here are some
+    examples showing our review culture in practice:
+    * link 1
+    * link 2
+    * link 3
+* Etc...
```

Comment:
Similar issue with this 'etc' - it in a list clearly labeled 'requirements', even if it is in a template.

Reply:
I agree that it'd be good to make this as easy to follow as possible.

In fact, ideally we'd restore and update the proposal template, and turn it into an issue template, so that people have a very easy to follow guide for how to apply